lrtable/src/lib/statetable.rs

```diff
@@ -478,21 +552,21 @@ fn resolve_shift_reduce<StorageT: 'static + Hash + PrimInt + Unsigned>(
     off: usize,
     tidx: TIdx<StorageT>,
     pidx: PIdx<StorageT>,
-    stidx: StIdx
-) -> u64
-where
+    stidx: StIdx, // State we want to shift to
+    shift_reduce: &mut Vec<(TIdx<StorageT>, PIdx<StorageT>, StIdx)>,
+    state: StIdx // State in which the conflict occured
```

Comment:
Ouch, this is a terrible name given `stidx` also exists! We have to come up with something that's not so much of a duplicate. Maybe `shift_stidx` and `conflict_stidx` or something like that?

Reply:
I changed it. But now I'm wondering if this is a similar problem and needs addressing: https://github.com/softdevteam/grmtools/blob/f8e5a1d9d8e58faf1a05dd370064ebb653548498/lrtable/src/lib/statetable.rs#L216

Reply:
That's less bad, because `stidx` is `STIdx` and `state` is an entirely different type.